src/components/common/Toolbar/Toolbar.tsx

```diff
@@ -62,53 +75,62 @@ export function Toolbar({ children, ...props }: ToolbarProps) {
     [focusedToolbarKey]
   );
 
-  const handleKeyDown = useCallback((event: KeyboardEvent) => {
-    const list = ref.current;
-    const currentFocus = document.activeElement;
+  const handleKeyDown = useCallback(
+    (event: KeyboardEvent) => {
+      const list = ref.current;
+      const currentFocus = document.activeElement;
 
-    // Keyboard interactions are based on
-    // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/toolbar_role#keyboard_interactions
-    switch (event.key) {
-      case 'ArrowLeft':
-        event.preventDefault();
-        moveFocus(list, currentFocus, previousItem);
-        break;
-      case 'ArrowRight':
-        event.preventDefault();
-        moveFocus(list, currentFocus, nextItem);
-        break;
-      case 'ArrowUp':
-        // Up and down arrow keys are used to navigate in a single radio group in a toolbar
-        // https://developer.mozilla.org/en-US/docs/web/accessibility/aria/roles/radiogroup_role#keyboard_interactions
-        if (isRadioInput(currentFocus)) {
+      // Keyboard interactions are based on
+      // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/toolbar_role#keyboard_interactions
+      switch (event.key) {
+        case 'ArrowLeft':
           event.preventDefault();
-          moveFocus(
-            findParentRadioGroup(currentFocus, list),
-            currentFocus,
-            previousItem
-          );
-        }
-        break;
-      case 'ArrowDown':
-        if (isRadioInput(currentFocus)) {
+          moveFocus(list, currentFocus, previousItem);
+          break;
+        case 'ArrowRight':
           event.preventDefault();
-          moveFocus(
-            findParentRadioGroup(currentFocus, list),
-            currentFocus,
-            nextItem
-          );
-        }
-        break;
-      case 'Home':
-        event.preventDefault();
-        moveFocus(list, null, nextItem);
-        break;
-      case 'End':
-        event.preventDefault();
-        moveFocus(list, null, previousItem);
-        break;
-    }
-  }, []);
+          moveFocus(list, currentFocus, nextItem);
+          break;
+        case 'ArrowUp':
+          // Up and down arrow keys are used to navigate in a single radio group in a toolbar
+          // https://developer.mozilla.org/en-US/docs/web/accessibility/aria/roles/radiogroup_role#keyboard_interactions
+          if (isRadioInput(currentFocus)) {
+            event.preventDefault();
+            moveFocus(
+              findParentRadioGroup(currentFocus, list),
+              currentFocus,
+              previousItem
+            );
+          } else if (toolbarDirection && toolbarDirection === 'column') {
```

Comment:
```suggestion
          } else if (toolbarDirection === 'column') {
```

If `toolbarDirection` is `undefined`, it can't be `"column"`. You also set the default value to `"row"` so it will never be `undefined`.